config/development.json

```diff
@@ -22,7 +22,7 @@
 	"logout_url": "https://wordpress.com/wp-login.php?action=logout&redirect_to=https%3A%2F%2F|subdomain|wordpress.com",
 	"facebook_api_key": "249643311490",
 	"dsp_stripe_pub_key": "pk_live_51LYYzQF53KN4RFN0oHFYHpzmHGpuUAXZ6ygDAFxM4XkRArRsET0orrofmsynkd9DhFaOGEsDZ3RC4f8mgI0zhEyN000X8i0Mqx",
-	"dsp_widget_js_src": "https://dsp.wp.com/widget.js",
+	"dsp_widget_js_src": "http://localhost:3004/widget.js",
```

Comment:
I think this change got pushed by accident, we need to change it to the correct one.

Reply:
whoops, I'll revert. Thanks